sdk/communication/communication-common/review/communication-common.api.md

```diff
@@ -37,7 +37,7 @@ export type CommunicationIdentifierKind = CommunicationUserKind | PhoneNumberKin
 // @public
 export interface CommunicationTokenCredential {
     dispose(): void;
-    getToken(abortSignal?: AbortSignalLike): Promise<AccessToken>;
+    getToken(abortSignalOrScopes?: AbortSignalLike | string | string[]): Promise<AccessToken>;
```

Comment:
I am a bit iffy on this. @DominikMe what do you think?

Reply:
Hi Orlando, we can't change the public CommunicationTokenCredential design to make implementation easier for us. 

Instead, we should add a helper to common that can adapt to the right internal format. This should work:

```typescript
export const createCommunicationTokenAuthPolicy = (
  credential: CommunicationTokenCredential
  ): RequestPolicyFactory => {
    return bearerTokenAuthenticationPolicy({
      getToken: (_scopes, options) => credential.getToken(options?.abortSignal)
  }, []);
}
```